### DISCUSSION TITLE
My PRs are not showing in the leaderboard today.

### DISCUSSION THREAD

**Discussion by ParnaRoyChowdhury777** [codeharborhub/codeharborhub.github.io#3510] [Q&A] [answered]
My projects are there in the website. The PRs are removed. Please check it. Only 1 PR is there.

here are my projects :
I have added QR scanner, Scientific calculator
![Screenshot 2024-07-17 072616](https://github.com/user-attachments/assets/ddbb6a7a-0863-4c4d-b350-3bdd21e0924f)

but the projects pr are not there : 

![Screenshot 2024-07-17 072711](https://github.com/user-attachments/assets/b8579217-9718-4344-add0-781da10264a2)

**Comment by ajay-dhangar** [comment, marked as answer]
Hello @ParnaRoyChowdhury777,

I want to address some concerns regarding the contributor scores for our project during GSSoC'24. Unfortunately, any contributions made before June 4th were not counted due to an issue with the project's queue being full. 

On June 4th, I made the necessary changes and set up the deployment for our site on GitHub. As a result, contributions made after June 4th are being properly counted towards your scores.

Thank you for your understanding and continued contributions!

Best regards,  
@Ajay-Dhangar

**Reply by ajay-dhangar** [reply]
And if you want to know more about it, you can ask @sanjay-kv. I think he will help you with this issue.

**Reply by sanjay-kv** [reply]
Hi @Ajay-Dhangar we need to fix this today
any relevant url link you can provide before june 4th

**Reply by ajay-dhangar** [reply]
Before June 4th, my repo URL was https://github.com/CodeHarborHub/codeharborhub. After June 4th, the URL was updated to https://github.com/CodeHarborHub/codeharborhub.github.io. Maybe this has something to do with it? I can only view data on the leaderboard of https://github.com/CodeHarborHub/codeharborhub.github.io but not view https://github.com/CodeHarborHub/codeharborhub

**Comment by ajay-dhangar** [comment]
Hey @ParnaRoyChowdhury777, please check out your issues and Q&A on https://github.com/GSSoC24/Contributor/discussions/505.

**Comment by ajay-dhangar** [comment]
Hi @ParnaRoyChowdhury777, to calculate the score for your leaderboard, you'll need to add something to those files and then try it out at the bottom. 

Here is the code you need to include for your author profile in the documentation:

```md
---

<h2>Authors:</h2>

<div style={{display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between', gap: '10px'}}>
{['github_username'].map(username => (
 <Author key={username} username={username} />
))}
</div>
```

and for showcase, you can reupload your project image and create PRs and mention prev... PRs link I will again merge with same levels

**Reply by ParnaRoyChowdhury777** [reply]
> Hi @ParnaRoyChowdhury777, to calculate the score for your leaderboard, you'll need to add something to those files and then try it out at the bottom. 
> 
> Here is the code you need to include for your author profile in the documentation:
> 
> ```md
> ---
> 
> <h2>Authors:</h2>
> 
> <div style={{display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between', gap: '10px'}}>
> {['github_username'].map(username => (
>  <Author key={username} username={username} />
> ))}
> </div>
> ```
> 
> and for showcase, you can reupload your project image and create PRs and mention prev... PRs link I will again merge with same levels

Ok....I am doing it..Thank you
